sdk/storage/tests_invoke.yml

```diff
@@ -3,7 +3,7 @@ parameters:
   InjectedPackages: ''
 
 jobs:
-  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
```

Comment:
Is this file necessary anymore? We removed the reference to tests_invoke.yml in the storage tests.yml IIRC.

Reply:
Aha. I'd prefer that `tests.yml` call into `tests_invoke.yml`. 

The reason that it was set up a bit non-standard was to minimize the duplication of the environment variables being passed to `archetype-sdk-tests`.

Reply:
fixed!